kerchunk/grib2.py

```diff
@@ -158,7 +158,10 @@ def scan_grib(
             if "typeOfLevel" in m and "level" in m:
                 name = m["typeOfLevel"]
                 data = np.array([m["level"]])
-                attrs = cfgrib.dataset.COORD_ATTRS[name]
+                try:
+                    attrs = cfgrib.dataset.COORD_ATTRS[name]
+                except KeyError:
+                    logger.warning(f"Couldn't find coord {name} in dataset")
```

Comment:
Suggest this is not as severe as a warning, it is (apparently) expected in some cases.

```suggestion
                    logger.debug(f"Couldn't find coord {name} in dataset")
```